### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [demokratie-live/democracy-client#1533] [Ideas]
Would a Mastodon account for this project be possible?

**Comment by lukasgabriel** [comment]
Possible, sure  :)  
I'm not certain that there would be a large enough audience for it to justify the effort, though.
Paging @visionmakery

**Comment by lukasgabriel** [comment]
@Lukas2112 
We discussed the possibility of hosting a Mastodon instance internally but decided that it's not one of our priorities right now.
Thank you for the suggestion though!
